### PR TITLE
Adds screen reader text to pagination - trello ticket #463

### PR DESCRIPTION
**Pull request by BNewing**
This adds text to the pagination links. I've added text to the previous and next links as well as the numbers.

### CODE REVIEW

File: web/app/themes/justicejobs/search-apply-page.php [hunk 1]
```diff
@@ -263,16 +263,16 @@ class="search_contain__radio search_contain__radio--map"
...
-
+            'base' => str_replace( $big, '%#%', get_pagenum_link( $big, false ) ),
+            'format' => '?paged=%#%',
+            'mid_size' => 2,
+            'current' => max( 1, get_query_var('paged') ),
+            'total' => $job_query->max_num_pages,
+            'prev_text' => '<span class="screen-reader-text"> Search results - previous page</span><span aria-hidden="true">PREV</span>',
+            'next_text' => '<span class="screen-reader-text"> Search results - </span>NEXT <span class="screen-reader-text">page</span>',
+            'before_page_number' => '<span class="screen-reader-text"> Search results - page </span>',
+            'after_page_number' => '<span class="screen-reader-text"> of ' . __( $job_query->max_num_pages ). '</span>'
```
Comment (wilson1000-MoJ) [inline web/app/themes/justicejobs/search-apply-page.php]: __() is a WP translation function used to display (if present) the text in a different language.
It is best practice to define a text-domain in the themes root style.css file and then pass the text-domain as a string in the second parameter in each function call. This is regarded as good code development as it essentially creates a theme translation layer and will make translation processes easier in future. 
Would you kindly add the text-domain?
Ref: https://developer.wordpress.org/reference/functions/__/

File: web/app/themes/justicejobs/search-apply-page.php [hunk 2]
```diff
@@ -388,30 +388,13 @@ class="map"
...
             'current' => max( 1, get_query_var('paged') ),
             'total' => $job_query->max_num_pages,
-            'prev_text' => 'PREV',
-            'next_text' => 'NEXT'
-        ) );
-
+            'prev_text' => '<span class="screen-reader-text"> Search results - previous page</span><span aria-hidden="true">PREV</span>',
+            'next_text' => '<span class="screen-reader-text"> Search results - </span>NEXT <span class="screen-reader-text">page</span>',
+            'before_page_number' => '<span class="screen-reader-text"> Search results - page </span>',
+            'after_page_number' => '<span class="screen-reader-text"> of ' . __( $job_query->max_num_pages ). '</span>'
```
Comment (wilson1000-MoJ) [inline web/app/themes/justicejobs/search-apply-page.php]: As above

File: web/app/themes/justicejobs/search.php [hunk 3]
```diff
@@ -582,8 +582,10 @@ class="search_contain__radio search_contain__radio--map"
                     'mid_size' => 2,
                     'current' => max(1, get_query_var('paged')),
                     'total' => $job_query->max_num_pages,
-                    'prev_text' => 'PREV',
-                    'next_text' => 'NEXT'
+                    'prev_text' => '<span class="screen-reader-text"> Search results - previous page</span><span aria-hidden="true">PREV</span>',
+                    'next_text' => '<span class="screen-reader-text"> Search results - </span>NEXT <span class="screen-reader-text">page</span>',
+                    'before_page_number' => '<span class="screen-reader-text"> Search results - page </span>',
+                    'after_page_number' => '<span class="screen-reader-text"> of ' . __( $job_query->max_num_pages ). '</span>'
```
Comment (wilson1000-MoJ) [inline web/app/themes/justicejobs/search.php]: Add text-domain to __()

File: web/app/themes/justicejobs/search.php [hunk 4]
```diff
@@ -721,29 +723,15 @@ class="map"
...
```
Comment (wilson1000-MoJ) [inline web/app/themes/justicejobs/search.php]: Add text-domain to __()

**Review by wilson1000-MoJ — CHANGES_REQUESTED** [review]

**Comment by BNewing** [comment]
Thank you for the change requests @wilson1000-MoJ ! I've now added another commit to address this feedback

**Pull request merged**